plugin/pkg/shoot/quotavalidator/admission.go

```diff
@@ -32,6 +39,24 @@ const (
 	PluginName = "ShootQuotaValidator"
 )
 
+var (
+	quotaMetricNames = [...]v1.ResourceName{
```

Comment:
`[...]` is needed?

Reply:
Nope. Set this array to fixed size as the count of its elements is always fixed.